documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -395,7 +569,28 @@ For an _Entity_ named “MySubAssetEntity”, the payload is minimized to the fo
 }
 ----
 
-For a BasicEventElement named “MyBasicEvent”, the payload is minimized to the following:
+If the _Entity_ itself is the subject for ValueOnly the serialization is minimized to the following:
+
+[source,json,linenums]
+----
+{
+    "statements": {
+      "MaxRotationSpeed": 5000
+    },
+    "entityType": "SelfManagedEntity",
```

Comment:
Is this really part of what is considered the **value** of an Entity? I would expect it to be considered metadata are therefore excluded from valueOnly serialization.